modules/policy-change-management.adoc

```diff
@@ -33,6 +33,20 @@ An authorized SRE reviewer must approve advancement to each step. The reviewer c
 
 Some changes are released to production incrementally, using feature flags to control availability of new features to specified clusters or customers.
 
+You can implement a maintenance exclusion by avoiding changes in {cluster-manager} for the following components:
+
+* Deleting a cluster
+* Adding or removing an identity provider
+* Adding or removing a user from an elevated group
+* Changing cluster networking visibility
+* Adding or removing machine pools
+* Defining upgrade policies
```

Comment:
This should be "Initiating an upgrade"... Defining an upgrade policy, such as changing from Automatic to Manual, is actually implied as part of the maintenance exclusion process in the call-out below.

Reply:
Thanks, Bill. I've updated the docs with the suggestion.